moduleframework/tools/dockerlint.py

```diff
@@ -97,6 +99,81 @@ def test_helpmd_is_present(self):
         self.assertTrue(self.dp.check_helpmd_is_present())
 
 
+class DockerfileLinterInContainer(container_avocado_test.ContainerAvocadoTest):
+    """
+    :avocado: enable
+
+    """
+
+    def _file_to_check(self, doc_file_list):
+        test_failed = False
+        for doc in doc_file_list.split('\n'):
+            exit_status = self.run("test -e %s" % doc, ignore_status=True).exit_status
+            if int(exit_status) == 0:
+                self.log.debug("%s doc file exists in container" % doc)
+                test_failed = True
+        return test_failed
+
+    def test_all_nodocs(self):
+        """
+        This test checks all docs installed in container. As delivered by Platform
+        as delivered by installing packages RUN command
```

Comment:
The part `As delivered by Platform as delivered by installing packages RUN command` doesn't make any sense to me. What were you trying to say?

Reply:
Some documentation can be installed by BASE image. Like FROM ....
Nothing more.

Reply:
Ok, I see. I don't think the quoted sentence says that, I'd recommend just removing it.